jwql/website/apps/jwql/monitor_pages/yaml/monitor_bias_interface.yaml

```diff
@@ -0,0 +1,336 @@
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
```

Comment:
There seems to be a lot of repetition in this.  Is there no way to make a more generalized interface file?  Perhaps a question for @gkanarek 

Not a dealbreaker for this PR, but if there is a way to improve this, we should look into it in the future.

Reply:
yeah definitely - originally I had a general bias vs time figure that I updated in a loop for each amp/odd/even region, but it ended up being ~twice as slow as separating them like what's in this PR. But yeah, I'm sure there's a better way.